MonstercatNet/Models/Playlist/PlaylistTrack.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace SoftThorn.MonstercatNet.Models.Playlist
```

Comment:
I'd like to cut down the namespace to the root namespace, so that users of this lib have to only import a single namespace to get all the available features.